packages/data-fetcher/src/transaction/transaction.service.ts

```diff
@@ -79,11 +82,24 @@ export class TransactionService {
       transactionReceipt
     );
 
+    const isEvmLike = transactionInfo.to === null;
+    const toAddress = isEvmLike ? "0x" : transactionInfo.to;
+    const contractAddress = isEvmLike
+      ? getCreateAddress({ from: transactionInfo.from, nonce: transactionInfo.nonce })
+      : "";
```

Comment:
Why empty string and not null?

Reply:
To align with etherscan where they are leaving empty string in cases like that one.